src/Console/Storage.php

```diff
@@ -33,7 +33,7 @@ class Storage extends \Asika\SimpleConsole\Console
 {
 	protected $helpOptions = ['h', 'help', '?'];
 
-	/** @var StorageManager */
+	/** @var \Friendica\Core\Storage\Repository\StorageManager */
```

Comment:
Extraneous absolute path
```suggestion
	/** @var StorageManager */
```